test/test_common/utility.h

```diff
@@ -357,8 +357,9 @@ class TestUtility {
    * @return a filename based on the process id and current time.
    */
 
-  static std::string uniqueFilename() {
-    return absl::StrCat(getpid(), "_", std::chrono::system_clock::now().time_since_epoch().count());
+  static std::string uniqueFilename(absl::string_view prefix = "") {
+    return absl::StrCat(prefix, getpid(), "_",
```

Comment:
Please add a `-` between prefix and getpid(), as both could be numbers and there could be ambiguity